<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindowClass</class>
 <widget class="QWidget" name="LoginWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konkon - Login</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="welcome_label">
     <property name="text">
      <string>Welcome to Konkon!</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="username_label">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="username_field"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="password_label">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="password_field"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="login_button">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="register_button">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="github_button">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="loginwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>register_button</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindowClass</receiver>
   <slot>on_RegisterButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login_button</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindowClass</receiver>
   <slot>on_LoginButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>github_button</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindowClass</receiver>
   <slot>github_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_RegisterButtonClick()</slot>
  <slot>on_LoginButtonClick()</slot>
  <slot>github_button_clicked()</slot>
 </slots>
</ui>
